pkg/component/autoscaling/vpa/vpa.go

```diff
@@ -231,6 +232,10 @@ func (v *vpa) emptyPodMonitor(name string) *monitoringv1.PodMonitor {
 	return &monitoringv1.PodMonitor{ObjectMeta: monitoringutils.ConfigObjectMeta(name, v.namespace, seed.Label)}
```

Comment:
Shouldn't we use `v.getPrometheusLabel()` for vpa-recommender's Pod monitor as well? Or is it something related to https://github.com/gardener/gardener/blob/cd0992b79d2c8bfa9d2c12a2e4a0dba47c10bc3f/pkg/component/autoscaling/vpa/recommender.go#L296-L298?

Reply:
Yeah, I think this label always says `seed`, because we _always_ scrape the recommenders with the seed prometheus. There, we have there an aggregated dashboard allowing us to compare the recommenders of the different shoots.